src/TwigComponent/src/ComponentRenderer.php

```diff
@@ -130,11 +130,10 @@ private function preRender(MountedComponent $mounted, array $context = []): PreR
 
             // add computed properties proxy
             ['computed' => new ComputedPropertiesProxy($component)],
-
+            $props,
             // add attributes
             [$metadata->getAttributesVar() => $mounted->getAttributes()],
-            $props,
-            ['__props' => $props]
+            ['__props' => array_merge($mounted->getInputProps(), $props)]
```

Comment:
This changes a lot the content of $variables (even for component not concerned by this change)

Before:
```php

  "outerScope" => []
  "this" => Symfony\UX\TwigComponent\Tests\Fixtures\Component\WithAttributes {#3194
    +prop: "prop value 1"
  }
  "computed" => Symfony\UX\TwigComponent\ComputedPropertiesProxy {#3224
    -cache: []
    -component: Symfony\UX\TwigComponent\Tests\Fixtures\Component\WithAttributes {#3194}
  }
  "attributes" => Symfony\UX\TwigComponent\ComponentAttributes {#3222
    -rendered: []
    -attributes: array:4 [
      "class" => "bar"
      "style" => "color:red;"
      "value" => ""
      "autofocus" => true
    ]
  }
  "prop" => "prop value 1"
  "__props" => array:1 [
    "prop" => "prop value 1"
  ]
]
```

After
```php
[
  "outerScope" => []
  "this" => Symfony\UX\TwigComponent\Tests\Fixtures\Component\WithAttributes {#3194
    +prop: "prop value 1"
  }
  "computed" => Symfony\UX\TwigComponent\ComputedPropertiesProxy {#3224
    -cache: []
    -component: Symfony\UX\TwigComponent\Tests\Fixtures\Component\WithAttributes {#3194}
  }
  "attributes" => Symfony\UX\TwigComponent\ComponentAttributes {#3222
    -rendered: []
    -attributes: array:4 [
      "class" => "bar"
      "style" => "color:red;"
      "value" => ""
      "autofocus" => true
    ]
  }
  "prop" => "prop value 1"
  "class" => "bar"
  "style" => "color:red;"
  "value" => ""
  "autofocus" => true
  "__props" => array:5 [
    "prop" => "prop value 1"
    "class" => "bar"
    "style" => "color:red;"
    "value" => ""
    "autofocus" => true
  ]
]
```

I know tests are green, but that seems to me really strange and not needed